pkg/addons/helm.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2025 The Kubernetes Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package addons
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"path"
+	"strings"
+
+	"github.com/pkg/errors"
+	"k8s.io/minikube/pkg/minikube/assets"
+	"k8s.io/minikube/pkg/minikube/command"
+	"k8s.io/minikube/pkg/minikube/vmpath"
+)
+
+func helmCommand(ctx context.Context, chart *assets.HelmChart, enable bool) *exec.Cmd {
+	var args []string
+
+	if !enable {
+		args = []string{
+			fmt.Sprintf("KUBECONFIG=%s", path.Join(vmpath.GuestPersistentDir, "kubeconfig")),
+			"helm", "uninstall", chart.Name,
+		}
+		if chart.Namespace != "" {
+			args = append(args, "--namespace", chart.Namespace)
+		}
+		return exec.CommandContext(ctx, "sudo", args...)
+	}
+
+	args = []string{
+		fmt.Sprintf("KUBECONFIG=%s", path.Join(vmpath.GuestPersistentDir, "kubeconfig")),
+		"helm", "upgrade", "--install", chart.Name, chart.Repo, "--create-namespace",
+	}
+	if chart.Namespace != "" {
+		args = append(args, "--namespace", chart.Namespace)
+	}
+	if chart.Values != nil {
+		for _, value := range chart.Values {
+			args = append(args, "--set", value)
+		}
+	}
+	if chart.ValueFiles != nil {
+		for _, value := range chart.ValueFiles {
+			args = append(args, "--values", value)
+		}
+	}
+	return exec.CommandContext(ctx, "sudo", args...)
+}
+
+func helmInstall(addon *assets.Addon, runner command.Runner) error {
+	_, err := runner.RunCmd(exec.Command("test", "-f", "/usr/bin/helm"))
+	if err != nil {
+		// If not, install it
```

Comment:
how about installing it through their Script ?
https://helm.sh/docs/intro/install#from-script